package/doc/sphinx/source/conf.py

```diff
@@ -14,6 +14,8 @@
 import sys
 import os
 import datetime
+sys.path.insert(0, os.path.abspath('../../..'))
```

Comment:
Could you explain why this is being added?

Reply:
Sorry for forgetting remove the line. 
I am developing on MacOS platform and thus the compiling of documentation with original code would raise an ModuleNotFoundError: `No module named 'MDAnalysis'`. My practice follows the instruction of latest documentation for developing. After adding the line in more upper position, `sphinx-build` could work.